classes/classes/Output.as

```diff
@@ -12,7 +12,17 @@ package classes
 		import classes.GlobalFlags.kFLAGS;
 		import coc.view.MainView;
 
-		public function Output() {}
+		private static var _instance:Output = new Output();
+
+		public function Output()
```

Comment:
I get your point, but I chose to throw an error, when people try this. 
"Error: Output can only be accessed through Output.init()" is more informative IMHO